src/AttributeTranslator/NullAttributeTranslator.php

```diff
@@ -7,7 +7,7 @@
 use Yiisoft\Validator\AttributeTranslatorInterface;
 
 /**
- * An attribute translator that returns attribute name as is.
+ * An attribute translator that returns attribute name as is without actually translating it.
  */
 final class NullAttributeTranslator implements AttributeTranslatorInterface
 {
```

Comment:
Method `translate()` is left without PHPDoc at all (see https://github.com/yiisoft/validator/pull/473/files#r1052958155 for details).

Reply:
That is alright. It should be inherited from an interface but isn't due to IDE bug.